### PR TITLE
#918 Fixing updated newsletter forms

### PR DESCRIPTION
**Pull request by thetwopct**
Work for #918

### CODE REVIEW

**Comment by cjyabraham** [comment]
**Event footer**
A big improvement on what we had. Why not just make all 4 input fields the same width? It looks odd that the country field is slightly narrower?

**Non-event footer**
LGTM

File: web/wp-content/themes/lfevents/src/assets/scss/modules/_event-footer.scss [hunk 1]
```diff
@@ -27,13 +27,12 @@
 .event-footer-newsletter {
   margin-bottom: 3rem;
 
-  .legal-consent-container {
```
Comment (cjyabraham) [inline web/wp-content/themes/lfevents/src/assets/scss/modules/_event-footer.scss:30]: Can you not delete this? From time-to-time the LF team adds legal text on these newsletter forms and it shows visible on the site unless we have this.

File: web/wp-content/themes/lfevents/src/assets/scss/modules/_non-event-footer.scss [hunk 2]
```diff
@@ -97,17 +100,16 @@
...
+          margin-top: 0.75rem;
+        }
+
         @media (min-width: 700px) {
           padding: 12px 24px;
           font-size: 14px;
           line-height: 14px;
         }
       }
-
```
Comment (cjyabraham) [inline web/wp-content/themes/lfevents/src/assets/scss/modules/_non-event-footer.scss:106]: Undo this change as well

Comment (cjyabraham) [inline web/wp-content/themes/lfevents/src/assets/scss/modules/_non-event-footer.scss:106]: Undo this change as well

File: web/wp-content/themes/lfevents/src/assets/scss/modules/_non-event-footer.scss [hunk 3]
```diff
@@ -97,17 +100,16 @@
...
-      .legal-consent-container {
-        display: none;
-      }
```
Comment (cjyabraham) [inline web/wp-content/themes/lfevents/src/assets/scss/modules/_non-event-footer.scss:109]: Undo this change as well

File: web/wp-content/themes/lfevents/src/assets/scss/modules/_event-footer.scss (hunk 1 above)
Comment (thetwopct) [reply web/wp-content/themes/lfevents/src/assets/scss/modules/_event-footer.scss:30]: Ah ok. Can you add a note to it? Just looks like random unused css otherwise.

**Comment by thetwopct** [comment]
Updated. Only included .legal-consent-container {} once, no need to have it twice.

**Review by cjyabraham — APPROVED** [review]
